docs/src/main/asciidoc/drools.adoc

```diff
@@ -331,8 +331,8 @@ When you have your Quarkus project configured, you can add the Drools Quarkus ex
 </dependency>
 
 <dependency>
-    <groupId>org.assertj</groupId>
-    <artifactId>assertj-core</artifactId>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-devtools-testing</artifactId>
```

Comment:
What made you think that, is it referenced somewhere? `quarkus-devtools-testing` is mostly internal and it doesn't really make sense to use it in a Quarkus application.